src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -498,5 +500,65 @@ private static void ValidateRoleDefinition(PSRoleDefinition roleDefinition)
                 throw new ArgumentException(ProjectResources.InvalidActions);
             }
         }
+        public static void ValidateScope(string scope)
+        {
+
+            if (scope.Length == 0 || !scope.StartsWith("/"))
+            {
+                throw new ArgumentException(string.Format(ProjectResources.ScopeShouldBeginWithSlash, scope));
+            }
+
+            // "/" is a valid scope.
+            if (string.Compare(scope, "/") == 0)
+            {
+                return;
+            }
+
+            var parts = scope.Substring(1).Split('/');   // Skip the leading '/'
+
+            if (parts.Contains(string.Empty))
```

Comment:
ARM does ignore the extra slash, but it is not clear to end user, and i don't think there is no benefit for end user to type double slash in the scope.

---
In reply to: [119178261](https://github.com/Azure/azure-powershell/pull/4019#discussion_r119178261) [](ancestors = 119178261)